src/locales/en.json

```diff
@@ -284,6 +284,13 @@
   "nft_details_page_title": "Writing NFT",
   "nft_details_page_title_collector": "Collectors",
   "nft_details_page_title_transfer": "Transfer Your NFT",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "Everyone is a content creator in this modern digital era. However different limitations still exist on internet platforms, there is no one place in which words and writings can be flown freely. Blocktrend, the most subscribed chinese blockchain media, recorded the story of the first bookstore that restore freedom and ownership to readers and authors. Welcome to the \"Web3 Bookstore\"",

```

Comment:
We are all content creators in this modern digital era. However, there isn't a place where words can be bought and sold freely beyond platform restrictions. Blocktrend, the most subscribed Chinese Blockchain media recorded the story of the first Web3 bookstore that restored freedom and ownership to readers and writers. The world's first bookstore where you can read freely and create independently. Please enter the "Unmanned Bookstore".